packages/unity-interface/kernelConfigForRenderer.ts

```diff
@@ -18,11 +15,7 @@ export function kernelConfigForRenderer(): KernelConfigForRenderer {
       nameValidCharacterRegex: nameValidCharacterRegex.toString().replace(/[/]/g, ''),
       nameValidRegex: nameValidRegex.toString().replace(/[/]/g, '')
     },
-    features: {
-      enableBuilderInWorld: false,
-      enableAvatarLODs: isFeatureEnabled(store.getState(), FeatureFlags.AVATAR_LODS, false),
-      enableExploreV2: false
-    },
+    features: {},
```

Comment:
This is a breaking change, it will generate an imcompatibility conflict between versions of the renderer that assume `enableAvatarLODs: true` exist and now kernel will send `avatar_lods: true`